### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by rdgoite**
Added a quick tip for Kind users to get LoadBalancers to work.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ $ minikube tunnel
 
 {{< /tip >}}
 
+{{< tip >}}
+If you are running Istio on Kind, follow the [guide for setting up MetalLB](https://kind.sigs.k8s.io/docs/user/loadbalancer/) to get `LoadBalancer` service type to work.
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md:51]:
```suggestion
If you are running Istio using Kind on Linux, follow the [guide for setting up MetalLB](https://kind.sigs.k8s.io/docs/user/loadbalancer/) to get `LoadBalancer` type services to work.
```

**Review by Arhell — COMMENTED** [review]
Please sign CLA

**Comment by rdgoite** [comment]
I have signed the CLA. Not sure why the system doesn't seem to recognise it. Are you able to confirm this?

**Comment by craigbox** [comment]
Done, with apologies, and thanks!

**Review by craigbox — APPROVED** [review]

**Pull request merged**